exchanges/deribit/deribit_wrapper.go

```diff
@@ -259,17 +256,31 @@ func (d *Deribit) FetchTradablePairs(ctx context.Context, assetType asset.Item)
 // them in the exchanges config
 func (d *Deribit) UpdateTradablePairs(ctx context.Context, forceUpdate bool) error {
 	assets := d.GetAssetTypes(false)
+	var wg sync.WaitGroup
```

Comment:
We have a beautiful new helper for this pattern:

```
        errs := common.CollectErrors(len(assets))
	for x := range assets {
		go func(x int) {
			defer errs.Wg.Done()
			pairs, err := d.FetchTradablePairs(ctx, assets[x])
			if err != nil {
				errs.C <- err
				return
			}
			err = d.UpdatePairs(pairs, assets[x], false, forceUpdate)
			if err != nil {
				errs.C <- err
				return
			}
		}(x)
	}
	return errs.Collect()
```

Reply:
That's great 🥹